pyproject.toml

```diff
@@ -16,6 +16,8 @@ hvac = "^0.11.2"
 rdflib = "^6.0.2"
 honeybadger = "^0.7.1"
 types-requests = "^2.26.1"
+folio-uuid = "^0.2.0"
```

Comment:
What is the purpose of this.  Why is it called a "uuid", but it doesn't look like a UUID?

Reply:
This generates module from Ebsco generates a deterministic uuid for FOLIO records based on the okapi_uri, a enumeration value based on the record type, and an ILS id, in our case the Sinopia BF Instance URL. 
```python
> from folio_uuid import FOLIONamespaces, FolioUUID

> okapi_uri = "https://okapi-folio.dev.edu/"
> instance_uri = "https://api.development.sinopia.io/resource/0000-1111-2222-3333"

> print(FolioUUID(okapi_url, FOLIONamespaces.instances, instance_uri))
98a0337a-ec22-53aa-8ffc-933a86d10159
```